specifyweb/frontend/js_src/lib/components/Attachments/index.tsx

```diff
@@ -72,7 +79,13 @@ export function AttachmentsView(): JSX.Element | null {
   );
 }
 
-function Attachments(): JSX.Element {
+function Attachments({
+  onClick,
+  displayScale = true,
```

Comment:
Instead of `displayScale`, could you call this `isEmbedded` or `isInDialog`?

Reply:
or actually, you don't even need to provide a prop. Instead you could just check if React.useContext(DialogContext) is not undefined - that will tell you that you are in a dialog